tests/test_coin_combination.py

```diff
@@ -1,3 +1,19 @@
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+def test_type():
+    assert type(get_coin_combination(0)) == list
```

Comment:
Btw, it's better to check type like this:
```
instance(get_coin_combination(0), list)
```

Reply:
Thanks for a tip.